composer.json

```diff
@@ -10,5 +10,8 @@
             "name": "Jitendra Adhikari",
             "email": "jiten.adhikary@gmail.com"
         }
-    ]
+    ],
+    "require-dev": {
+            "phpunit/phpunit": "~4.0"
```

Comment:
yup, but again with a tilde (~), we can still fetch next significant release (< 5.0.0), so it is basically 4.5 as of now